test/docfx.Test/docfx.Test.csproj

```diff
@@ -4,20 +4,11 @@
     <TargetFramework>netcoreapp2.0</TargetFramework>
     <IsPackable>false</IsPackable>
     <LangVersion>Latest</LangVersion>
-    <CodeAnalysisRuleSet>../../stylecop.ruleset</CodeAnalysisRuleSet>
```

Comment:
>ruleset [](start = 40, length = 7)

does it mean no styles for Tests? Why?

Reply:
tests tends to generate a lot of stylecop/fxcop warnings. see [https://ci.appveyor.com/project/DocFXService/docfx/build/1.0.114](https://ci.appveyor.com/project/DocFXService/docfx/build/1.0.114)

---
In reply to: [181612693](https://github.com/dotnet/docfx/pull/2634#discussion_r181612693) [](ancestors = 181612693)